packages/SwingSet/misc-tools/store-stats/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "store-stats",
```

Comment:
let's name it `swingset-store-vatstats`, and add a `private: true` field just to remind us to give it a better name (something under `@agoric/`, probably) before we publish it.